src/components/TimeSince.tsx

```diff
@@ -0,0 +1,36 @@
+/*
+Copyright 2018 - 2022 The Alephium Authors
+This file is part of the alephium project.
+
+The library is free software: you can redistribute it and/or modify
+it under the terms of the GNU Lesser General Public License as published by
+the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+
+The library is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+GNU Lesser General Public License for more details.
+
+You should have received a copy of the GNU Lesser General Public License
+along with the library. If not, see <http://www.gnu.org/licenses/>.
+*/
+
+import dayjs from 'dayjs'
+import styled from 'styled-components'
+
+interface TimeSinceProps {
+  timestamp: number
+  className?: string
+  faded?: boolean
+}
+
+const TimeSince = ({ timestamp, className }: TimeSinceProps) => {
+  return <div className={className}>{dayjs(timestamp).fromNow()}</div>
+}
+
+const TimeSinceStyled = styled(TimeSince)`
+  ${({ faded, theme }) => (faded ? `color: ${theme.font.secondary}` : '')}
+`
+
+export default TimeSinceStyled
```

Comment:
Can be written simpler using our convention:

```suggestion
export default styled(TimeSince)`
  ${({ faded, theme }) => (faded ? `color: ${theme.font.secondary}` : '')}
`

Reply:
Note this immediate exporting only works for this particular set of components (`styled(Component)`) this will fail the linter rules if trying to immediately export any arrow-function components, i.e. `export default () => ...`

Reply:
yes, of course